src/victory-util/perf.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Memoizes multi-argument functions.
+ *
+ * NOTE: If `fn` receives another function as its argument, memoization is
+ * not guaranteed to work.  The function will be strigified, but it will
+ * no longer be a closure.
+ *
+ * @param  {Function}  fn  The function to memoize
+ *
+ * @return {Function}      Memoized `fn`.
+ */
+export const memoize = function memoize(fn) {
```

Comment:
Also, tests would be nice and should be super easy to write. Let's make sure we cover our base and corner cases!